### DISCUSSION TITLE
Test: 0.1.0 in Docker on GL.iNet MT6000

### DISCUSSION THREAD

**Discussion by richb-hanover** [Zoxc/crusader#43] [Wi-Fi routers and access points]
Client: Crusader GUI 0.1 (pre-built) running on macOS 12.7.5
Server: Crusader server 0.1 running in Docker container on router (GL.iNet MT6000) with stock firmware
Connection(s): Ethernet between MacBook and router LAN port
Observations: mediocre speeds, bad latency

![plot 2024-08-21 21 39 47](https://github.com/user-attachments/assets/9933b5f9-6840-48eb-a58d-e82cd1856aa9)

Thoughts?

**Comment by Zoxc** [comment]
It should be a faster than that. Is Docker causing trouble?

**Reply by richb-hanover** [reply]
I also was surprised. I have some other test cases that I will check out and create a careful report. Thanks

**Reply by dtaht** [reply]
Is there a qdisc on the container?

**Reply by richb-hanover** [reply]
See #55
